src/app/pages/LendingPage/components/CurrencyContainer/CurrencyRow.tsx

```diff
@@ -81,6 +82,7 @@ const CurrencyRow: React.FC<Props> = ({
             className="tw-block tw-w-full tw-rounded-lg"
             textClassName="tw-text-base"
             disabled={isEmptyBalance || withdrawLocked || !connected}
+            data-action-id={`lend-withdrawal-button-${asset}`}
```

Comment:
```suggestion
            data-action-id={`lend-withdrawalButton-${asset}`}
```
This was defined incorrectly on the spec